examples/b2c/checkout/integrations/classes/B2CDeliverySample.cls

```diff
@@ -2,22 +2,32 @@
 // in order to be processed by the checkout flow for the "Shipping" integration
 
 global class B2CDeliverySample implements sfdc_checkout.CartShippingCharges {
-
+    // You MUST change this to be your service or you must launch your own Heroku Service
+    // and add the host in Setup | Security | Remote site settings.
```

Comment:
What is this comment mean  exactly? Is there a default behavior that would make the default store work out of the box?

Reply:
The store should work out of the box because we are delegating the request to a static method that returns a mocked response.